### DISCUSSION TITLE
Issue with Enter Key Not Working in RustDesk Android Client When Connecting to Servers

### DISCUSSION THREAD

**Discussion by yfxycjtf** [rustdesk/rustdesk#10152] [Q&A] [answered]
I am encountering an issue when using the RustDesk Android client to connect to my Windows and Ubuntu servers. Specifically, the Enter key does not function properly after establishing the connection. Regardless of which server I connect to, the Enter key seems to be disabled, preventing me from executing commands or operations that require pressing the Enter key.

我在使用RustDesk安卓客户端连接我的Windows和Ubuntu服务器时遇到了一个问题，即回车键在连接后无法正常工作。无论是连接到哪个服务器，回车键都似乎被禁用了，导致我无法执行需要回车键的命令或操作。

**Comment by rustdesk** [comment, marked as answer]
Yes, it does not work on some input method, e.g. some baidu android input versions, updating your input version or changing it may work. We also added a help key for this, ready in nightly 1.3.4. https://github.com/rustdesk/rustdesk/pull/10068

![image](https://github.com/user-attachments/assets/d7a2f2c4-1c5e-4d54-b9fb-867e4a4f5eae)

**Reply by yfxycjtf** [reply]
Thank you very much for your timely and helpful response! Indeed, I had been using Baidu Input Method previously because of its good support for Wubi New Century. After switching to another input method, the Enter key is now functioning properly.

Your suggestion, as well as the addition of the help key in the 1.3.4 nightly version, are both very thoughtful. It's great to know that the RustDesk team is actively addressing such issues and continuously enhancing the user experience.
